src/Compilers/CSharp/Portable/BoundTree/BoundDecisionDagNode.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+using Roslyn.Utilities;
+
+#nullable enable
```

Comment:
the convention adopted in much of the nullable-annotated code already in Roslyn is to put the #nullable directive after the license and before any usings. e.g. [https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Symbols/MethodKind.cs](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Symbols/MethodKind.cs)